ci/jenkins/bin/cache-tests.sh

```diff
@@ -58,9 +57,10 @@ ${PREFIX}/bin/trafficserver stop
 
 # Now run it again, maybe, with the tweaked configs
 if [ "" != "$TWEAK" ]; then
-#    echo "CONFIG proxy.config.http.cache.required_headers INT 1" >> $RECORDS
-    echo "CONFIG proxy.config.http.negative_caching_enabled INT 1" >> $RECORDS
     ${PREFIX}/bin/trafficserver start
+    sleep 5
+    ${PREFIX}/bin/traffic_ctl config set proxy.config.http.negative_caching_enabled 1
+    sleep 5
```

Comment:
This sleep here isn't very exciting.  If we installed `yq` to the ci image you could do: `yq -i '.ts.http.negative_caching_enabled=1' $RECORDS` before starting traffic server. Or if traffic_ctl could do the out-of-band edit.

Reply:
True. 

traffic_ctl will take care of that. I’ll link this to the other pr tomorrow.  And update this line . 
thanks